stripe-patrons-data/src/test/scala/com/gu/patrons/services/SupporterDataDynamoServiceSpec.scala

```diff
@@ -0,0 +1,37 @@
+package com.gu.patrons.services
+
+import com.gu.supporterdata.model.Stage.DEV
+import com.gu.supporterdata.services.SupporterDataDynamoService
+import com.gu.test.tags.annotations.IntegrationTest
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+@IntegrationTest
+class SupporterDataDynamoServiceSpec extends AsyncFlatSpec with Matchers {
+  "SupporterDataDynamoService" should "return false if a subscription doesn't exist" in {
+    val identityId = "non_existent_identity_id"
+    val subscriptionId = "non_existent_subscription_id"
+    val dynamoService = SupporterDataDynamoService(DEV)
+    dynamoService.subscriptionExists(identityId, subscriptionId).flatMap {
+      case Right(userExists) if userExists =>
+        fail(s"Subscription $subscriptionId should not exist")
+      case Right(_) =>
+        succeed
+      case Left(errorMessage) => fail(errorMessage)
+    }
+  }
+  "SupporterDataDynamoService" should "return true if a subscription does exist" ignore {
+    // This test relies on a record being present in the dev data store so
+    // I'm marking it as ignored in case that record gets deleted
```

Comment:
What does the ignored test mean? Is it that we won’t be notified of a test failure, or something less strong than that?

Reply:
Ah, I looked it up: the test will always report as Ignored. Why have the test if it’ll always be ignored?

Reply:
It just means that it won't run unless the `ignored` keyword is switched back to `in`, so it won't be run by the scheduled integration test runner. It's arguable whether I should just delete it if it's not going to be run regularly but I like to have tests around which allow me to run individual bits of code easily so on balance I think it's worth keeping.